mozetl/cli.py

```diff
@@ -15,6 +16,7 @@ def entry_point():
 entry_point.add_command(churn.main, "churn")
 entry_point.add_command(clientsdaily.main, "clients_daily")
 entry_point.add_command(experimentsdaily.main, "experiments_daily")
+entry_point.add_command(retention.main, "retention")
```

Comment:
So this is a great example where the magic star import in the `mozetl.engagement.retention` is confusing. Just import the main function from the job module instead of relying on the start import. So this should be either:

```python
from mozetl.engagement.churn import job as churn_job
from mozetl.engagement.retention import job as retention_job
# ..
entry_point.add_command(churn_job.main, "churn")
# ..
entry_point.add_command(retention_job.main, "retention")
```

Or explicitly import the main function (and whatever you deem a callable that you want to expose in the new packages) in `mozetl/engagement/retention/__init__.py`:

```python
from .job import main  # noqa
```

and `mozetl/engagement/retention/__init__.py`:

```python
from .job import main  # noqa
```

I personally like the first better since for a submodule several levels deeps it's kind of easy to forget to maintain the API of the callables that exposed as API there.

Reply:
I went ahead with the first suggestion, I think I want to be explicit and straightforward with these changes.